### PR TITLE
Added Troubleshooting Entry for x509 Error

### LINKED ISSUE #232: Unable to pull images receive an X509 error
@rbankston commented on [Fri Feb 26 2021](https://github.com/astronomer/docs-archived/issues/270)

**Request type:**

- [ ] **New document:**: Add the `newcontent` label to this issue. 
- [X] **Significant change to an existing document**: Add the `enhancement` label to this issue. 

**What is the purpose of the document/enhancement? Select all that apply:**

- [ ] Document a new feature for an upcoming release. Add the appropriate release version label to this issue. 
- [ ] Information for an existing feature that isn't yet documented.
- [x] Additional considerations/ reference material for an existing feature or document.

**Problem/feature that requires documentation:**

Astronomer Registry fails to authenticate for pulling a new image with the error

```
Warning Failed 24m (x4 over 26m) kubelet, <node> Failed to pull image "registry.astronomer.base.domain/testing/airflow:deploy-5": rpc error: code = Unknown desc = Error response from daemon: Get https://registry.astronomer.base.domain/v2/: x509: certificate signed by unknown authority
```

On occasion the Houston pods and registry pods shared certificates get out of sync. To resolve this issue restart the Houston pods and then the registry pod. Restarting the failed pod will allow it to be created immediately.

  
**Are you the right person to review a draft of the document for technical accuracy? If not, who is?**

Yes

**Questions**
Not sure if this should be added to https://www.astronomer.io/docs/enterprise/v0.23/troubleshoot/debug-install or if we should place self service troubleshooting documents elsewhere

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #232

### CODE REVIEW

**Review by rbankston — APPROVED** [review]
Looks good to me

File: software/debug-install.md [hunk 1]
```diff
@@ -149,5 +149,13 @@ $ kubectl create secret generic astronomer-bootstrap --from-literal connection="
 secret/astronomer-bootstrap created
 $ kubectl delete --all pods --namespace <namespace>
 ```
-
 Restarting the pods will force them to pick up the new value.
+
+## Houston and Astronomer Software x509 Certificate Signed by Unknown Authority error
+
+Occasionally, the shared Houston and Astronomer Software registry pod certificates fail to synchronize and the following error message appears:
```
Comment (jwitz) [inline software/debug-install.md:156]:
```suggestion
Occasionally, the shared Houston and Astronomer Software registry Pod certificates fail to synchronize and the following error message appears: 
```

File: software/debug-install.md [hunk 2]
```diff
@@ -149,5 +149,13 @@ $ kubectl create secret generic astronomer-bootstrap --from-literal connection="
...
 Restarting the pods will force them to pick up the new value.
+
+## Houston and Astronomer Software x509 Certificate Signed by Unknown Authority error
+
+Occasionally, the shared Houston and Astronomer Software registry pod certificates fail to synchronize and the following error message appears: 
+
+```
+Warning Failed 24m (x4 over 26m) kubelet, <node> Failed to pull image "registry.astronomer.base.domain/testing/airflow:deploy-5": rpc error: code = Unknown desc = Error response from daemon: Get https://registry.astronomer.base.domain/v2/: x509: certificate signed by unknown authority
+```
+To resolve this issue and clear the error message, restart the Houston pods and then the Astronomer Software registry pod.
```
Comment (jwitz) [inline software/debug-install.md:161]:
```suggestion
To resolve this issue and clear the error message, restart the Houston pods and then the Astronomer Software registry Pod.
```

File: software/debug-install.md (hunk 1 above)
Comment (jwitz) [reply software/debug-install.md:156]: Capitalize Pod per https://kubernetes.io/docs/concepts/workloads/pods/

**Review by jwitz — APPROVED** [review]
One minor edit, otherwise LGTM!

**Pull request merged**